lib/quandl/api_config.rb

```diff
@@ -0,0 +1,42 @@
+module Quandl
+  class ApiConfig
+    API_KEY_THREAD_KEY = 'quandl_api_key'
+    API_BASE_THREAD_KEY = 'quandl_api_base'
+    API_VERSION_THREAD_KEY = 'quandl_api_version_key'
+    DEFAULT_PER_PAGE_THREAD_KEY = 'default_per_page'
+
+    class << self
+      def api_key=(api_key)
+        Thread.current[API_KEY_THREAD_KEY] = api_key
+      end
+
+      def api_key
+        Thread.current[API_KEY_THREAD_KEY]
+      end
+
+      def api_base=(api_base)
+        Thread.current[API_BASE_THREAD_KEY] = api_base
+      end
+
+      def api_base
+        Thread.current[API_BASE_THREAD_KEY] || 'https://www.quandl.com/api/v3'
+      end
+
+      def api_version=(api_version)
+        Thread.current[API_VERSION_THREAD_KEY] = api_version
+      end
+
+      def api_version
+        Thread.current[API_VERSION_THREAD_KEY]
+      end
+
+      def default_per_page=(per_page)
+        Thread.current[DEFAULT_PER_PAGE_THREAD_KEY] = per_page
+      end
+
+      def default_per_page
+        Thread.current[DEFAULT_PER_PAGE_THREAD_KEY] || 25
```

Comment:
remove this and make sure elasticsearch and kaminari's pagination's per page are consistent